content/en/security_platform/application_security/getting_started/nodejs.md

```diff
@@ -0,0 +1,136 @@
+---
+title: NodeJS Applications
+kind: documentation
+code_lang: nodejs
+type: multi-code-lang
+code_lang_weight: 50
+further_reading:
+    - link: 'https://github.com/DataDog/dd-trace-js'
+      tag: 'GitHub'
+      text: 'NodeJS Datadog Library source code'
+---
+
+{{% appsec-getstarted %}}
+
```

Comment:
```suggestion
- Your deployment environment is [compatible][1] with our NodeJS library.

[1]: /security_platform/application_security/setup_and_configure/?code-lang=go#supported-frameworks
```

Reply:
@Julio-Guerra Nice catch, I forgot the Compatibility link for a couple languages. Adding now.